rholang/src/main/scala/coop/rchain/rholang/interpreter/accounting/Costs.scala

```diff
@@ -93,6 +93,48 @@ trait Costs {
   // + then it copies all elements of the Par
   def toByteArrayCost[T <: StacksafeMessage[_]](a: T): Cost =
     Cost(ProtoM.serializedSize(a).value, "to byte array")
+
+  // Converting rholang BigInt, String or ByteArray into a GInt.
+  // The cost is estimated as the number of bytes in converted data.
+  def toIntCost(bi: BigInt): Cost  = Cost(bigIntSize(bi), "bigint to int")
+  def toIntCost(str: String): Cost = Cost(str.length, "string to int")
+
+  // TODO: Adjust the BigInt operation cost
```

Comment:
How can we know what to _adjust_? Is is not correct now?

Reply:
I forgot to delete comment. Thanks!